docs/iot/guides/update-control-api.md

```diff
@@ -0,0 +1,142 @@
+
+## Prerequisites
+
+To enable your application to use the update control functionality:
+
+1. To use the update control API at applications, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+```xml
+   <privileges>
+      <privilege>http://tizen.org/privilege/updatecontrol.admin</privilege>
+   </privileges>
+```
+
+2. To use the functions and data types of the update control API, include the `<update_control.h>` header file in your application
+
+3. If some initialization is required to use update control functions, you can use `update_control_initialize()`
+
+```cpp
+int error_code;
+
+error_code = update_control_initialize();
+if (error_code != UPDATE_CONTROL_ERROR_NONE)
+  return;
+```
+Likewise, there is a function `update_control_deinitialize()` for deinitialization.
+
+## Connecting to Server
+
+The new version of platform is usually uploaded at the server and each device should connect to server to get the new version.
+In some cases, the procedure about the account may be required for connection. It can be done by `update_control_initialize()` or some of following procedures.
+
+## Checking the Latest Version
+
+The device can query the latest version of current platform to server. It can be implemented simply by:
+```cpp
+int error_code;
+
+error_code = update_control_check_new_version();
+if (error_code != UPDATE_CONTROL_ERROR_NONE)
+  return;
+```
+
+If the newer version of platform is available, the value of `UPDATE_CONTROL_PROPERTY_UPDATE_AVAILABLE` will be set properly
+(It's type and value are depend on platform developer).
+The application can get the result through the `update_control_get_property()` function:
+```cpp
+int error_code;
+bool is_available;
+
+error_code = update_control_get_property(UPDATE_CONTROL_PROPERTY_UPDATE_AVAILABLE, (void **)&is_available);
+if (error_code == UPDATE_CONTROL_ERROR_NONE) {
+  if (is_available)
+    // New version platform is available
+    ...
+}
+```
+
+## Downloading and Updating to New Version
+
+If the newer version is available, the application can download it and update to the found one.
+The functions `update_control_download_package()` and `update_control_do_update()` are offered and their usage is similar to `update_control_get_property()`:
+```cpp
+int error_code;
+
+error_code = update_control_download_package();
+if (error_code != UPDATE_CONTROL_ERROR_NONE)
+  return;
+
+error_code = update_control_do_update();
+if (error_code != UPDATE_CONTROL_ERROR_NONE)
+  return;
+```
+Usually, downloading and updating are processed sequentially but the application also can call each function separately for intending operation.
+
+## Making an Update Reservation
+
+The device especially IoT may be critical at it's operation time. Because updating procedure may include unavailable time (ex. rebooting), it needs to schedule software update at specific time.
```

Comment:
The device especially IoT might be critical and its operation time. Because updating procedure might include unavailable time (For example,  rebooting), it needs to schedule software update at specific time. 

Question: What will happen to the operation time?

Reply:
To make it clear, it has been modified. Please check the latest one.